frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -2692,4 +2694,18 @@ private void createModRequest(ResourceRequest request, ResourceResponse response
             response.setProperty(ResourceResponse.HTTP_STATUS_CODE, "500");
         }
     }
+
+    private Set<String> getProjectGroups(List<Project> projects) {
+        Set<String> businessUnit = new HashSet<String>();
+        try {
+            if(!CommonUtils.isNullOrEmptyCollection(projects)) {
```

Comment:
Maybe this can be used `CommonUtils.isNotEmpty(projects)`

Reply:
Fixed please check.